fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/rescheduleloan/data/LoanRescheduleRequestDataValidator.java

```diff
@@ -156,15 +231,6 @@ public void validateForCreateAction(final JsonCommand jsonCommand, final Loan lo
                             .failWithCode("repayment.schedule.installment.does.not.exist", "Repayment schedule installment does not exist");
                 }
             }
-
-        }
```

Comment:
Why do you remove this validation?

Reply:
Not removed, just moved out of the method because of it's common with progressive loan